traffic_ops/traffic_ops_golang/crconfig/handler.go

```diff
@@ -273,7 +273,7 @@ func SnapshotOldGUIHandler(w http.ResponseWriter, r *http.Request) {
 	// We never store tm_path, even though low API versions show it in responses.
 	crConfig, err := Make(inf.Tx.Tx, cdn, inf.User.UserName, r.Host, inf.Config.Version, inf.Config.CRConfigUseRequestHost, false)
 	if err != nil {
-		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, errors.New(r.RemoteAddr+" making CRConfig: "+err.Error()))
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusBadRequest, errors.New(r.RemoteAddr+" making CRConfig: "+err.Error()), nil)
 		return
 	}
```

Comment:
An error returned from `crconfig.Make()` *is* 500-level, not 400-level. The endpoint should check at an earlier point whether a CDN with the given name exists and return a 400 bad request at *that* point if it doesn't.

Reply:
Ah sorry I misunderstood your comment.